ghedesigner/radial_numerical_borehole.py

```diff
@@ -8,12 +9,14 @@
 from ghedesigner.constants import TWO_PI
 
 
-class RadialCellType(object):
-    FLUID = 1
-    CONVECTION = 2
-    PIPE = 3
-    GROUT = 4
-    SOIL = 5
+class CellProps(IntEnum):
+    R_IN = 0
+    R_CENTER = auto()
+    R_OUT = auto()
+    K = auto()
+    RHO_CP = auto()
+    TEMP = auto()
+    VOL = auto()
```

Comment:
`RadialCellType` was unused so I removed it. Here's the new enum class that can be used to associate cell properties with their physical meaning.